packages/core/upload/admin/src/components/AssetDialog/BrowseStep/index.js

```diff
@@ -68,6 +82,8 @@ export const BrowseStep = ({
   selectedAssets,
 }) => {
   const { formatMessage } = useIntl();
+  const [view, setView] = usePersistentState(`${toUpper(pluginId)}_MODAL_VIEW`, viewOptions.GRID);
```

Comment:
I'd prefer if we'd prefix the key with `STRAPI_`

Reply:
Also: could we build it in a way so that `list` is the default and we only persist if it was changed to grid?

Reply:
should list be the default view? seeing as today we only have the grid view?